DungeonMaster.Data/Models/Model.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DungeonMaster.Data.Models
+{
+
+    public abstract class Model : BaseModel, IModel
+    {
+        public virtual int Id { get; set; }
+    }
+
+    public abstract class BaseModel
+    {
+    }
+
+    public interface IModel
+    {
+        int Id { get; set; }
```

Comment:
Any chance we could move basemodel and imodel to separate files?

Reply:
I left them in the same file since there wasn't a lot of code to begin with, but I don't see and issue with moving them. Do the same for Repositories and Services?